src/sentry/static/sentry/app/components/quickTrace/index.tsx

```diff
@@ -151,6 +158,47 @@ export default function QuickTrace({
     />
   );
 
+  if (traceLength === 1) {
+    nodes.push(
+      <Projects
+        key="missing-services"
+        orgId={organization.slug}
+        slugs={[current.project_slug]}
+      >
+        {({projects}) => {
+          const project = projects.find(p => p.slug === current.project_slug);
+          if (project) {
+            if (BACKEND_PLATFORMS.includes(project?.platform as string))
```

Comment:
Can we add in the `{}`?

Reply:
@Zylphrex sorry not sure what you mean by add in the `{}`?

Reply:
You're missing the `{}` around the body of the `if` statement

Reply:
🤦 ah, yeah definitely can add it in, 🤔 thought the lint would've have opinions on it